src/V8Platform.h

```diff
@@ -59,16 +59,11 @@ class ETWTracingController : public v8::TracingController {
       unsigned int flags,
       int64_t timestamp) override;
 
-  void UpdateTraceEventDuration(
-      const uint8_t *category_enabled_flag,

```

Comment:
> co [](start = 5, length = 3)

(nit) usually, having separate line per argument for long arguments is more readable

Reply:
This is an automatic formatting. We better keep it "as-is".